Dockerfile

```diff
@@ -8,8 +8,12 @@ ARG INSPEC_VERSION
 
 RUN apk update && \
     apk upgrade && \
-    apk --no-cache add python2 py-pip py-setuptools ca-certificates ruby ruby-rdoc ruby-irb openssh-client && \
-    apk --no-cache add --virtual .sd-build-dependencies gcc libffi-dev openssl-dev build-base ruby-dev python2-dev linux-headers musl-dev
+    apk --no-cache add python3 ca-certificates ruby ruby-rdoc ruby-irb openssh-client && \
+    apk --no-cache add --virtual .sd-build-dependencies gcc libffi-dev openssl-dev build-base ruby-dev python3-dev linux-headers musl-dev
+
+# Configure python3
+RUN pip3 install --upgrade pip setuptools && \
```

Comment:
Is pip then a symlink to pip3?
Perhaps you have to change the other usage(s) of `pip` beyond, too?

Reply:
Yeah, I checked it and pip refers to the pip version of python3
"pip 19.0.3 from /usr/lib/python3.6/site-packages/pip (python 3.6)"